README.md

```diff
@@ -0,0 +1,94 @@
+# go-billy-gluster [![GoDoc](https://godoc.org/gopkg.in/src-d/go-billy-gluster.v0?status.svg)](https://godoc.org/github.com/src-d/go-billy-gluster) [![Build Status](https://travis-ci.org/src-d/go-billy-gluster.svg)](https://travis-ci.org/src-d/go-billy-gluster) [![codecov.io](https://codecov.io/github/src-d/go-billy-gluster/coverage.svg)](https://codecov.io/github/src-d/go-billy-gluster)
+
+This package provides a basic [go-billy](https://github.com/src-d/go-billy) driver to access [gluster](https://www.gluster.org/) volumes. It uses libgfapi so it's not needed to mount the volume locally.
+
+Currently only basic actions are available, that is:
```

Comment:
maybe just
> So far, following actions are implemented:

Reply:
Somehow I tried to say that only `billy.Basic` interface is fully implemented but is not clear. I've changed it and added another action from `billy.Dir` that is implemented.